src/encoded/static/components/top_hits/multi/constants.js

```diff
@@ -0,0 +1,14 @@
+import TopHitsQuery, {
+    CollectionsQuery,
```

Comment:
I was hoping it would make it more apparent which query is which in this list:

<img width="614" alt="Screen Shot 2021-05-20 at 10 12 21 AM" src="https://user-images.githubusercontent.com/16053922/119021494-58044b80-b954-11eb-88a2-1420b2a53303.png">


I didn't want to change the actual name since other things still rely on it outside of the multisearch context.

Reply:
I was thinking if you changed the name within '../query' to `TopHitsResults` that’s still accurate regarldess of who relies on it; it’s just a more specific name than it used to be, but no less accurate. I’m not sure I’m covering all cases of this though, but that was my impression.

Reply:
>I was thinking if you changed the name within '../query' to TopHitsResults

Ah, yeah that makes sense. This also brings up other issue of aliasing the top hits results component and the data collection results component in same way.

I think maybe the problem in general is that the organizational structure is not quite right:

<img width="1299" alt="Screen Shot 2021-05-20 at 2 18 48 PM" src="https://user-images.githubusercontent.com/16053922/119050599-f0abc300-b976-11eb-97ec-f6713aff314a.png">

Originally it made sense that everything was in `top_hits` folder, and the `query` was simply the top hits query. But with new multisearch the top hits is just one possible section.

I've stuck most of the functionality in the `multi` folder mainly for lack of anywhere else to put it. I agree this is not ideal.